### DISCUSSION TITLE
Node Version Manager (NVM)

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#74] [General]
# Node Version Manager (NVM)

Check if Node Version Manager (NVM) is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/7ac91d43-f5fb-4c25-9a52-66a47d910881

**Comment by milkeryildirim** [comment]
I cannot find the nvm windows arm version. Could you please provide some details on how and where you found it?
